definition/data/sheets/AuthorisationCaseEvent.json

```diff
@@ -245,5 +245,6 @@
   {"CaseTypeID": "MoneyClaimCase", "CaseEventID": "ResumeClaimPaymentCitizen", "UserRole": "citizen", "CRUD": "CR"},
   {"CaseTypeID": "MoneyClaimCase", "CaseEventID": "ResumeClaimPaymentCitizen", "UserRole": "caseworker-cmc", "CRUD": "C"},
   {"CaseTypeID": "MoneyClaimCase", "CaseEventID": "CreateCitizenClaim", "UserRole": "citizen", "CRUD": "CR"},
-  {"CaseTypeID": "MoneyClaimCase", "CaseEventID": "CreateCitizenClaim", "UserRole": "caseworker-cmc", "CRUD": "CR"}
+  {"CaseTypeID": "MoneyClaimCase", "CaseEventID": "CreateCitizenClaim", "UserRole": "caseworker-cmc", "CRUD": "CR"},
+  {"CaseTypeID": "MoneyClaimCase", "CaseEventID": "ResendPin", "UserRole": "caseworker-cmc", "CRUD": "CR"}
```

Comment:
I suggest `ResetPin` instead, as `Resend` sounds like sending the same one again

Reply:
i agree, i thought that too but just followed what it says in the ticket. I guess we should put whatever makes more sense for case worker, i am guessing who ever created this ticket would have thought about that. What do others think?

Reply:
I agree with @andrewwa-kainos. Seeing as the event Id is not shown to the end user I would think that we should be ok to decide on the name of this within the team. Happy to be overruled though!

Reply:
agree, done.